### PR TITLE
Simplify grib process

### PR DESCRIPTION
**Pull request by zackarno**
Now that the API call grid param is fixed, I removed the regridding/interpolation step that was previously required

### CODE REVIEW

File: data-raw/02_process_mars_gribs_to_COGS.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        da_mean = da_mean.rio.write_crs("EPSG:4326", inplace=False)
+
+        for pub_date in pub_dates:
+            da_out = da_mean.sel(time=pub_date)
+            date_str = pd.to_datetime(pub_date).strftime("%Y-%m-%d")
+            filename_prefix = f"seas5_mars_tprate_em_{date_str}"
+            print(filename_prefix)
+
+            fms = da_out.forecastMonth.values
+            for i, fm_tmp in enumerate(fms):
```
Comment (t-downing) [inline data-raw/02_process_mars_gribs_to_COGS.py:73]: can just do `for fm_tmp in fms:`?

File: data-raw/02_process_mars_gribs_to_COGS.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        ds = ds.drop_vars("tprate")
+        da = ds["precip"]
+        da_mean = da.mean(dim=["number"])
+
+        # loop through pub dates & COGS
+        pub_dates = da_mean.time.values
+        forecast_months = da_mean.forecastMonth.values
+
+        # give a CRS
+        da_mean = da_mean.rio.write_crs("EPSG:4326", inplace=False)
```
Comment (t-downing) [inline data-raw/02_process_mars_gribs_to_COGS.py:64]: Yeah this is the better way to do this, I've found that `inplace=True` doesn't actually work.

**Review by t-downing — APPROVED** [review]
Looks good! Glad we solved the regridding issue. 

Random thing- I noticed there are some negative values in the forecast, particularly in months with low rainfall. Fine to leave them in for now, but might want to decide in the future how to deal with these.

File: data-raw/02_process_mars_gribs_to_COGS.py (hunk 1 above)
Comment (zackarno) [reply data-raw/02_process_mars_gribs_to_COGS.py]: good call! I think the `enumerate` process was left over from different code where I was merging multiple bands into one tif and using the `i` to specify the band position... but yea no necessary here any more as leadtimes are getting split by file. Addressed here: [0674cdb](https://github.com/OCHA-DAP/ds-aa-afg-drought/pull/4/commits/0674cdba1490a3e4708c4c3a92ddf1ac392890f0)

File: data-raw/02_process_mars_gribs_to_COGS.py (hunk 2 above)
Comment (zackarno) [reply data-raw/02_process_mars_gribs_to_COGS.py:64]: ha good to know it doesn't actually work! maybe should make an issue of it in rio?

**Comment by zackarno** [comment]
Hey thanks alot for the review!

Good observation about the negative values! Will have to figure something out for that. I've integrated the other comments/suggestions.

**Pull request merged**